embedded/drivers/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 project(DriversLib VERSION 0.1 LANGUAGES CXX)
 
-add_library(drivers src/i2c.cpp src/serial.cpp)
+add_library(drivers src/i2c.cpp src/serial.cpp src/analog.cpp)
```

Comment:
Can we possibly use a glob or something here to glob all *.cpp?

Reply:
It doesn't seem like this will just work. Did some googling around and it looks like this isn't supported too well by CMake?

Reply:
Sounds good, just a thought